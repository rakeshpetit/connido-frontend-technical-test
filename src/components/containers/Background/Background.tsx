import React, { Component } from 'react';
import { View, Image, StyleSheet } from 'react-native';

class Background extends Component {
  render() {
    const backgroundImage = require('../../../../assets/homeBg.png');
    return (
    <View style={styles.container}>
       <Image source={backgroundImage} />
    </View>
    );
  }
}

const styles = StyleSheet.create({
  container: {
    backgroundColor: '#ccc',
    opacity: 0.4,
    flex: 1,
    position: 'absolute',
    justifyContent: 'center'
  }
});

export default Background;
