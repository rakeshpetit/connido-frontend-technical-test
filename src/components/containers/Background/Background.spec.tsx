import React from 'react';
import renderer from 'react-test-renderer';

import Background from './Background';

it('renders Background correctly', () => {
  const tree = renderer.create(<Background />).toJSON();
  expect(tree).toMatchSnapshot();
});