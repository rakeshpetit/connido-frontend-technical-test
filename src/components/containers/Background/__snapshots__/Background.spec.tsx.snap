// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when there are no items 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#ccc",
      "flex": 1,
      "justifyContent": "center",
      "opacity": 0.4,
      "position": "absolute",
    }
  }
>
  <Image
    source={1}
  />
</View>
`;
